doc/content/gateways/dragino-lps8/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Dragino LPS8"
+description: ""
+weight: 
+---
+
+The **Dragino LPS8** is a LoRaWAN indoor gateway. Its operating system is based on the OpenWrt Linux system. The hardware includes an SX1308 LoRa concentrator that provides 10 programmable parallel demodulation paths. The gateway has an ethernet and WiFi backhaul. The Dragino LPS8 uses the Semtech legacy packet forwarder. The web-based management console allows you to configure and quickly connect the gateway with any LoRaWAN network. LPS8 supports EU868, IN865, US915, AU915, AS923, and KR920 ISM bands.
+
+This page guides you on how to connect the Dragino LPS8 to {{% tts %}}.
+
+![Dragino LPS8](dragino-lps8.png)
+
+## Prerequisites
+
+1. A user account on {{% tts %}} with user rights to add gateways.
+2. LPS8 connected to the internet and also your computer and the LPS8 gateway should be in the same network (E.g. your home WiFi network).
+3. The latest version of firmware running on the Dragino LPS8 (read the Dragino LPS8 [user manual](https://www.dragino.com/downloads/downloads/LoRa_Gateway/LPS8/LPS8_LoRaWAN_Gateway_User_Manual_v1.2.0.pdf) for instructions on firmware upgrade)
+
+## Gateway Settings on the LPS8
+
+Let’s get started with configuring the Dragino LPS8 LoRaWAN gateway using the web-based gateway configuration portal.
+
+1. Open a browser.
+2. Go to the address bar and enter the IP address along with the port number (the default port number is 8000) of your gateway (E.g. `192.168.1.2:8000`) and then press **Enter**.
+3. A window will pop-up for a username and password. The default username is `root` and the password is `dragino` unless someone has changed the login credentials and then click **OK**.
+4. Once you logged in, you will navigate to the home page of the configuration portal that is a web-based GUI.
+5. On the portal page, on the top menu, select **LoRaWAN > LoRaWAN**.
+
+![Dragino LPS8 portal](dragino-lps8-portal.png)
+
+6. Now set the values for the **Server Settings** as shown and described here.
+
+- **LoRaWAN Service Provider**: choose **Custom** from the drop-down list.
+- **Server address**: provide your LoRaWAN gateway server address. You can find it on {{% tts %}} **Gateway Overview** page (see the next section).
+- **Latitude and Longitude**: provide the geolocation of your gateway (optional).
+- **Email**: provide your email address (optional).
+- Keep the **Gateway ID**, **default server port upstream** and **server port downstream**, and other settings.
+
+7. Select **Save & Apply**.
+
+![Dragino LPS8 LoRaWAN config](lps8-lorawan-config.png)
+
+## Gateway Settings on {{% tts %}}
```

Comment:
Replace this section with "Register the Gateway on TTS", see here https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/doc/content/gateways/mikrotikrouterboard/_index.md#registration

Also move it before "configure the gateway"

Reply:
ok